packages/checkout/sdk/src/api/http/httpClient.ts

```diff
@@ -0,0 +1,110 @@
+import axios, { AxiosInstance, AxiosRequestConfig, AxiosResponse } from 'axios';
+import { CheckoutModuleConfiguration } from '../../types';
+import { CheckoutError, CheckoutErrorType } from '../../errors';
+
+const PUBLISHABLE_KEY_PREFIX = 'pk_imapik-';
+
+const publishableKeyWhitelistedDomain = [
```

Comment:
can we use the term `allowlist` here, thats what we use throughout checkout

Reply:
Yep. good call. done